README.md

```diff
@@ -165,21 +166,21 @@ brave-browser> npm run sync --create branch_name
 
 ### Update the current branch to latest remote
 ```bash
-brave-core> git pull
+brave-core> git pull # or git reset --hard origin/branch_name
```

Comment:
these are not equivalent unless you fetch before reset

Reply:
true! I'll leave out, this is not a `git` readme after all

Reply:
I just dislike doing pull when it's not the `master` branch

Reply:
I never use `git pull`, I always fetch/merge/reset/...